docs/src/pages/[platform]/theming/style-props/examples/SizeExamples.tsx

```diff
@@ -0,0 +1,33 @@
+import { Badge, useTheme } from '@aws-amplify/ui-react';
+
+//use any CSS value
+export const SizeStylePropExample = () => {
+  return (
+    <Badge variation="info" width="100%" height="2rem">
```

Comment:
Can we make the Badge not take up 100% of the width? Looks too long
<img width="1208" alt="Screen Shot 2022-06-18 at 1 24 40 PM" src="https://user-images.githubusercontent.com/48109584/174451650-5ba247ef-9306-4030-bfa1-381fc32ca309.png">

Reply:
+1

Reply:
I think this may be because of a flex container? if you wrap the example in an extra div, I think it won't do that.

Reply:
I've updated the size example to use a card instead of a badge because the issue was that we are using design tokens which gives us very limited size options and the next biggest size of 4.5em wasn't big enough.  But with a card the fontsize can be set in a surrounding view and then reset on the displaying text component so that setting the width using the em value is large enough for the contained text.

Reply:
as per scotts suggestion this has been changed to a view with a background color